Gemfile

```diff
@@ -51,9 +51,11 @@ gem 'webpacker', '~> 5.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.crailrom/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'active_storage_base64'
```

Comment:
we should add a fixed version, you can check the one installed in the `Gemfile.lock`